.github/workflows/misspell.yml

```
@@ -18,8 +18,8 @@ jobs:
       - uses: peter-evans/create-pull-request@v5
         with:
           token: ${{ secrets.PRBOT_PAT }}
-          commit-message: "chore: fixes by cosmos-sdk bot"
-          title: "chore: typos fixes by cosmos-sdk bot"
+          commit-message: "chore: fixed by cosmos-sdk bot"
+          title: "chore: typos fixed by cosmos-sdk bot"
```

```suggestion
          title: "chore: fix spelling errors"
```

what do you think of these instead of whats there

---

lgtm